src/frontend/js/components/CourseProductCourseRuns/index.spec.tsx

```diff
@@ -0,0 +1,372 @@
+      // still be disabled and a message should inform user that he/she cannot enroll now.
+      $button = screen.getByRole('button', {
+        name: `Enrollment will open on ${dateFormatter.format(
+          new Date(courseRun.enrollment_start),
+        )}`,
+      });
+      expect($button.disabled).toBe(true);
+    });
+  });
+
+  describe('EnrolledCourseRun', () => {
+    const Wrapper = ({ code, children }: PropsWithChildren<{ code: string }>) => (
+      <IntlProvider locale="en">
+        <CourseCodeProvider code={code}>
+          <QueryClientProvider client={createQueryClient()}>
+            <JoanieApiProvider>{children}</JoanieApiProvider>
+          </QueryClientProvider>
+        </CourseCodeProvider>
+      </IntlProvider>
+    );
+
+    it('renders enrollment information', () => {
+      const enrollment: Enrollment = JoanieEnrollmentFactory.generate();
+      fetchMock.get('https://joanie.test/api/courses/00000/', 200);
+      render(
+        <Wrapper code="00000">
+          <EnrolledCourseRun enrollment={enrollment} />
+        </Wrapper>,
+      );
+
+      // - It should render course dates,
+      screen.getByText(dateFormatter.format(new Date(enrollment.start)), {
+        selector: '.course-runs-item__date',
+      });
+      screen.getByText(dateFormatter.format(new Date(enrollment.end)), {
+        selector: '.course-runs-item__date',
+      });
+
+      // - ... a link to access to the course,
+      const $link: HTMLLinkElement = screen.getByRole('link', { name: 'Go to course' });
+      expect($link).toHaveAttribute('href', enrollment.resource_link);
+
+      // - ... a call to action to unroll to the course.
+      screen.getByRole('button', { name: 'Unroll' });
+    });
+
+    it('allows to unroll', async () => {
+      const course: Course = CourseFactory.generate();
+      const enrollment: Enrollment = JoanieEnrollmentFactory.generate();
+      fetchMock.get(`https://joanie.test/api/courses/${course.code}/`, 200);
+
+      render(
+        <Wrapper code={course.code}>
+          <EnrolledCourseRun enrollment={enrollment} />
+        </Wrapper>,
+      );
+
+      const $button: HTMLButtonElement = screen.getByRole('button', { name: 'Unroll' });
+
+      fetchMock.resetHistory();
+      const enrollmentDeferred = new Deferred();
+      fetchMock.put(
+        `https://joanie.test/api/enrollments/${enrollment.id}/`,
+        enrollmentDeferred.promise,
+      );
+
+      await act(async () => {
+        fireEvent.click($button);
+      });
+
+      // - While request is pending, an accessible spinner should be displayed
+      screen.getByRole('status', { name: 'Unrolling...' });
+
+      await act(async () => {
+        enrollmentDeferred.resolve(200);
+      });
+
+      const calls = fetchMock.calls();
+      expect(calls).toHaveLength(2);
+      // A request to unroll user should have been executed
+      expect(calls[0][0]).toBe(`https://joanie.test/api/enrollments/${enrollment.id}/`);
```

Comment:
```suggestion
      expect(calls[0][0]).toBe(`https://joanie.test/api/enrollments/${enrollment.id}/`);
```
ditto expected params